src/components/AdvancedSearch.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import './AccomodationSearch.css'
+import Box from '@mui/material/Box';
+import InputAdornment from '@mui/material/InputAdornment';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+import { Hotel, KingBed, Person } from '@mui/icons-material';
+import { DatePicker, LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { MDBBtn } from 'mdb-react-ui-kit';
+
+interface State {
+    location: string;
+    date1: Date;
+    date2: Date;
+    personCount: number;
+    accType: Object
+}
+
+const AdvancedSearch = () => {
```

Comment:
Add state to form and `onSubmit` handler

Reply:
Be mindful when adding state to the form to keep the form state as an object instead of creating a lot of `useState` calls for each of the form's fields.
Refer to this link for help and an example. Let me know if you get stuck.
https://www.w3schools.com/react/react_forms.asp#:~:text=Multiple%20Input%20Fields